terraform/variables.tf

```diff
@@ -0,0 +1,98 @@
+variable "resource_group_name" {
+  description = "Name of the Resource Group for the projects"
```

Comment:
Doar ca FYI, daca alegem sa punem valorile direct in main aici o sa ramana doar variabilele a caror valoare va veni din afara teraform-ului, cum ar fi secretele